examples/pipelines/hf_dataset_pipeline/hf_dataset_pipeline.py

```diff
@@ -0,0 +1,44 @@
+"""
+This module includes a reusable component import and a basic pipeline utilising it.
+"""
+
+import json
+
+from kfp import components as comp
+from kfp import dsl
+
+from config.general_config import GeneralConfig, KubeflowConfig
+from helpers.upload import compile_and_upload_pipeline
+
+# Load Component
+load_from_hub_op = comp.load_component(
+    '/Users/nielsrogge/Documents/python_projects/express/examples/pipelines/hf_dataset_pipeline/components/load_from_hub/component.yaml')
+
+run_id = '{{pod.name}}'
```

Comment:
This won't work since it's not materialized, Could you add a TODO or a ticket for this and replace with a default value?

Reply:
What do you mean with this? How should this value be substituted?

Reply:
It should be defined inside the pipeline function for it to take place otherwise you get an error. Haven't dug deep into it yet. Can also use `kfp.dsl.RUN_ID_PLACEHOLDER` somewhere inside Express 

![Screenshot from 2023-03-15 16-59-47](https://user-images.githubusercontent.com/47530815/225371917-3bff4079-f06b-4d70-a18e-dfd40b458f8f.png)

Reply:
Ok great!